pkg/injector/config.go

```diff
@@ -14,6 +14,7 @@ type Config struct {
 	SidecarImage           string `envconfig:"SIDECAR_IMAGE" required:"true"`
 	SidecarImagePullPolicy string `envconfig:"SIDECAR_IMAGE_PULL_POLICY"`
 	Namespace              string `envconfig:"NAMESPACE" required:"true"`
+	KubeClusterDomain      string `envconfig:"KUBE_CLUSTER_DOMAIN" ignored:"true"`
```

Comment:
/cc @yaron2 I am concerned about adding K8s specific information here. On the other hand, it is unlikely that the injector is applicable outside K8s. Thoughts?

Reply:
That's fine, the injector is k8s specific